webviz_ert/controllers/ensemble_selector_controller.py

```diff
@@ -81,14 +88,21 @@ def set_callback(
                 if op["value"] not in selected_ens_value
             ]:
                 ensemble_selection_store["selected"].remove(element)
+            parent.save_state(
+                "ensembles",
+                [el["label"] for el in ensemble_selection_store["selected"]],
+            )
+
         if triggered_id == parent.uuid(f"ensemble-refresh-button"):
             parent.clear_ensembles()
             refresh_data(project_id=parent.project_identifier)
+            parent.save_state("ensembles", [])
```

Comment:
This causes a reset of all selection in the current view, while the other views keep their state. This seems a bit arbitrary from a user perspective.

Reply:
I would rather expect the save state feature to keep the state also of the current view, so that a refresh would only update the lists and plots, not the selections in the saved state.